connectors/mcp_base.py

```
@@ -2,14 +2,13 @@
 # Universal context connection for all external systems
 
 from abc import ABC, abstractmethod
-from typing import Dict, List, Any, Optional
-import json
-import asyncio
+from typing import Dict
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)